website/content/docs/concepts/host-discovery/index.mdx

```diff
@@ -58,5 +58,6 @@ Boundary currently supports dynamic host catalog for AWS and
 Azure and we will continue to grow this ecosystem to support additional providers.
 
 You can get started with dynamic host catalogs for AWS
-[here](/boundary/tutorials/host-management/aws-host-catalogs)
-and for Azure [here](/boundary/tutorials/host-management/azure-host-catalogs).
+[here](/boundary/tutorials/host-management/aws-host-catalogs),
+ for Azure [here](/boundary/tutorials/host-management/azure-host-catalogs)
+ and GCP [here](/boundary/tutorials/host-management/gcp-host-catalogs).
```

Comment:
```suggestion

To get started with dynamic host catalogs, refer to the following topics:

- [AWS dynamic host catalogs](/boundary/docs/concepts/host-discovery/aws)
- [Azure dynamic host catalogs](/boundary/docs/concepts/host-discovery/azure)
- [GCP dynamic host catalogs](/boundary/docs/concepts/host-discovery/gcp)
```

Reply:
We won't have a tutorial yet for GCP when this is released, so I'm thinking we can add a **Next steps** heading above and refer users to the documentation topics instead. This is the format we have been using for our newer content.